### PR TITLE
feat: Add providers and apply static pages

### PR DESCRIPTION
**Pull request by whDongRui**

### CODE REVIEW

File: src/components/Layout/SideNav/navMap.js [hunk 1]
```diff
@@ -34,14 +34,24 @@ export const subNavMap = {
...
         link: '/dashboard/applications',
         active: 'applications'
       },
-      { name: '合约', link: '#', active: '#' },
-      { name: '保证金', link: '#', active: '#' }
+      {
+        name: '合约',
+        link: '#',
+        active: '#',
+        isDisabled: true
```
Comment (iwisunny) [inline src/components/Layout/SideNav/navMap.js]: `disabled` is better than `isDisabled`

File: src/components/Layout/SideNav/navMap.js [hunk 2]
```diff
@@ -34,14 +34,24 @@ export const subNavMap = {
...
+      },
+      {
+        name: '保证金',
```
Comment (iwisunny) [inline src/components/Layout/SideNav/navMap.js]: Maybe translation can be added ahead of time

File: src/components/Layout/index.jsx [hunk 3]
```diff
@@ -86,12 +88,14 @@ class Layout extends Component {
...
       pageTitle,
       isCenterPage,
+      centerWidth,
       hasBack
     } = this.props;
 
     const { isNormal, isDev, isAdmin } = this.props.user;
     const hasMenu = (isDev || isAdmin) && !isHome;
     const hasSubNav = hasMenu && !noSubMenu && !hasBack;
+    const maxWidth = isCenterPage || hasBack ? `${centerWidth}px` : '100%';
```
Comment (iwisunny) [inline src/components/Layout/index.jsx]: refactor `maxWidth` to className is better

File: src/pages/Dashboard/Providers/Applications/index.jsx [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+import { observer, inject } from 'mobx-react';
+import { Link } from 'react-router-dom';
+import { translate } from 'react-i18next';
+import classnames from 'classnames';
+
+import { Table } from 'components/Base';
+import Layout from 'components/Layout';
+import Status from 'components/Status';
+
+import providers from '../mock/applications';
```
Comment (iwisunny) [inline src/pages/Dashboard/Providers/Applications/index.jsx]: If `providers` is mocked, should add `// todo`, then will be refactored

File: src/pages/Dashboard/Providers/Applications/index.jsx [hunk 5]
```diff
@@ -0,0 +1,139 @@
...
+  { name: 'Unprocessed', value: 'unprocessed' }
+];
+
+@translate()
+@inject(({ rootStore }) => ({
+  rootStore
+}))
+@observer
+export default class Applications extends Component {
+  constructor(props) {
```
Comment (iwisunny) [inline src/pages/Dashboard/Providers/Applications/index.jsx]: No need constructor, 
```
this.state={..}
```
is more compact

File: src/components/Layout/index.jsx (hunk 3 above)
Comment (whDongRui) [reply src/components/Layout/index.jsx]: There is for set center page max width， similar Dialog component props width.

File: src/pages/Dashboard/Providers/Applications/index.jsx [hunk 6]
```diff
@@ -0,0 +1,139 @@
...
+      activeType: 'processed'
+    };
+  }
+
+  async componentDidMount() {}
+
+  changeType = type => {
+    const { activeType } = this.state;
+
+    if (type !== activeType) {
```
[the remark this answers is not in the document]
Comment (whDongRui) [reply src/pages/Dashboard/Providers/Applications/index.jsx]: If no check , there will maybe send unnecessary request when the active type is click type.

File: src/pages/Dashboard/Providers/CertificateInfo/index.jsx [hunk 7]
```diff
@@ -0,0 +1,66 @@
...
+          </dl>
+          <dl>
+            <dt>{t('公司官网')}</dt>
+            <dd>http://www.crancloud.com/</dd>
+          </dl>
+          <dl>
+            <dt>{t('业务简介')}</dt>
+            <dd>
+              <pre>
+                云基众智—成就边缘加速新价值，提供部署在企业侧的物端边缘计算产品众智云基站和部署在云端的边缘加速服务众智云。
```
[the remark this answers is not in the document]
Comment (whDongRui) [reply src/pages/Dashboard/Providers/CertificateInfo/index.jsx]: The data of api ,there is mock data for show style.

File: src/pages/Dashboard/Providers/Detail/index.jsx [hunk 8]
```diff
@@ -0,0 +1,221 @@
...
+@inject(({ rootStore }) => ({
+  rootStore,
+  appStore: rootStore.appStore
+}))
+@observer
+export default class ProviderDetail extends Component {
+  async componentDidMount() {
+    // todo
+    const { appStore } = this.props;
+    appStore.fetchAll({ status: 'active' });
```
[the remark this answers is not in the document]
Comment (whDongRui) [reply src/pages/Dashboard/Providers/Detail/index.jsx]: ok

File: src/components/Layout/SideNav/navMap.js (hunk 1 above)
Comment (whDongRui) [reply src/components/Layout/SideNav/navMap.js]: ok

File: src/pages/Dashboard/Providers/Applications/index.scss [hunk 9]
```diff
@@ -0,0 +1,43 @@
+@import '~scss/vars';
+
+.types {
+  margin-bottom: 16px;
+
+  > label {
```
[the remark this answers is not in the document]
Comment (whDongRui) [reply src/pages/Dashboard/Providers/Applications/index.scss]: scss grammar should support  > in {}.

File: src/pages/Dashboard/Providers/Applications/index.jsx (hunk 6 above)
[the remark this answers is not in the document]
Comment (iwisunny) [reply src/pages/Dashboard/Providers/Applications/index.jsx]: If state not changed, component will not render

**Pull request merged**